src/rgw/rgw_acl_s3.cc

```diff
@@ -477,6 +477,10 @@ int RGWAccessControlPolicy_S3::create_from_headers(RGWUserCtl *user_ctl, const R
 int RGWAccessControlPolicy_S3::rebuild(RGWUserCtl *user_ctl, ACLOwner *owner, RGWAccessControlPolicy& dest,
                                        std::string &err_msg)
 {
+  #ifdef WITH_JAEGER
+    span_structure ss;
+    start_trace(std::move(ss), {}, s, "rgw_acl_s3.cc RGWAccessControlPolicy_S3::rebuild", true);
```

Comment:
better to use compiler macros for the file and function names

Reply:
@yuvalif I did not this could you please give an example?

Reply:
g++ has these macros: https://gcc.gnu.org/onlinedocs/cpp/Standard-Predefined-Macros.html
for c++ functions to be nicely demangeled use: https://gcc.gnu.org/onlinedocs/gcc/Function-Names.html
however, need to make sure that this would also work for clang++

Reply:
noted